cli/src/commands/content/updateChannel.ts

```diff
@@ -0,0 +1,89 @@
+import ContentDirectoryCommandBase from '../../base/ContentDirectoryCommandBase'
+import { IOFlags, getInputJson, saveOutputJson } from '../../helpers/InputOutput'
+import { NewAsset} from '@joystream/types/content'
+import {ChannelMetadata} from '@joystream/content-metadata-protobuf'
+import { Vec, Option} from '@polkadot/types';
+import AccountId from '@polkadot/types/generic/AccountId';
+import { Bytes } from '@polkadot/types/primitive';
+
+type ChannelUpdateParametersInput = {
+  assets: Option<Vec<NewAsset>>,
+  new_meta: ChannelMetadata.AsObject,
+  reward_account: Option<AccountId>,
+}
+
+type ChannelUpdateParameters = {
+  assets: Option<Vec<NewAsset>>,
+  new_meta: Bytes,
+  reward_account: Option<AccountId>,
+}
+
+export default class UpdateChannelCommand extends ContentDirectoryCommandBase {
+  static description = 'Update existing content directory channel.'
+  static flags = {
+    context: ContentDirectoryCommandBase.contextFlag,
+    ...IOFlags,
+  }
+
+  static args = [
+    {
+      name: 'channelId',
+      required: true,
+      description: 'ID of the Channel',
+    },
+  ]
+
+  async run() {
+
+    let { context, input, output } = this.parse(UpdateChannelCommand).flags
+
+    const { channelId } = this.parse(UpdateChannelCommand).args
+
+    if (!context) {
+      context = await this.promptForContext()
+    }
+
+    const currentAccount = await this.getRequiredSelectedAccount()
+    await this.requestAccountDecoding(currentAccount)
+
+    const actor = await this.getActor(context)
```

Comment:
Good point, did not thought about that before, but it would not possible to convert owner into actor, because channel owner `Curator` representation does not hold `curator_id` to perform authorization. https://github.com/Joystream/joystream/blob/sumer/runtime-modules/content/src/permissions/mod.rs#L153